$OpenBSD: patch-Source_WebKit2_Platform_CoreIPC_unix_ConnectionUnix_cpp,v 1.1 2013/04/13 09:11:02 landry Exp $
Source/WebKit2/Platform/CoreIPC/unix/ConnectionUnix.cpp:317: error: elements of array 'iovec iov [1]' have incomplete type
Source/WebKit2/Platform/CoreIPC/unix/ConnectionUnix.cpp:317: error: storage size of 'iov' isn't known
Source/WebKit2/Platform/CoreIPC/unix/ConnectionUnix.cpp:481: error: elements of array 'iovec iov [3]' have incomplete type
Source/WebKit2/Platform/CoreIPC/unix/ConnectionUnix.cpp:481: error: storage size of 'iov' isn't known
--- Source/WebKit2/Platform/CoreIPC/unix/ConnectionUnix.cpp.orig	Fri Feb  1 11:46:15 2013
+++ Source/WebKit2/Platform/CoreIPC/unix/ConnectionUnix.cpp	Fri Mar  8 00:57:25 2013
@@ -31,6 +31,7 @@
 #include "DataReference.h"
 #include "SharedMemory.h"
 #include <sys/socket.h>
+#include <sys/uio.h>
 #include <unistd.h>
 #include <errno.h>
 #include <fcntl.h>
